site/_includes/footer.html

```diff
@@ -28,13 +28,13 @@ <h5>{{ site.footer.title }}</h5>
                 </ul>
             </div>
             <div class="col-12 col-md-4 text-center text-md-right">
-              <a href="/" alt="Homepage"><img src="/img/{{ site.logo }}" class="logo" /></a>
+              <a href="/" aria-label="Octant homepage"><img src="/img/{{ site.logo }}" class="logo" alt="Homepage"/></a>
```

Comment:
This should say Velero homepage, not Octant :)

Reply:
Ruined by copy / paste :p